ZeeguuAPI/ZeeguuAPI/ZeeguuAPIDefaults.swift

```diff
@@ -84,23 +108,33 @@ extension ZeeguuAPI {
 	
 	func sendAsynchronousRequest(request: NSURLRequest, completion: (response: String?, error: NSError?) -> Void) {
 		let session = NSURLSession.sharedSession()
-		NSLog("Sending request for url \"%@\": %@\n\n", request.URL!, request);
+		if (self.enableDubugOutput) {
```

Comment:
there's at least one NSLog(...) which is outside of the enableDebug guard. 
And there's multiple of them which are of the form if (debug) { NSLog } . 
Could you maybe define a ZLog function that would eliminate the redundant checks? :)